src/backend/updates.rs

```diff
@@ -64,12 +71,60 @@ fn post_upgrade() {
         || spawn(State::finalize_upgrade()),
     );
 
+    migrate_posts();
+
     // post upgrade logic goes here
     set_timer(Duration::from_millis(0), move || {
         spawn(post_upgrade_fixtures());
     });
 }
 
+fn migrate_posts() {
```

Comment:
Just to double check: we still have the old end-point that is callable by stalwarts that can trigger this function manually, right?

Reply:
Yes! This is the `archive` endpoint I mention in the description!